bucket_30/gnatcross/specification

```diff
@@ -140,6 +147,22 @@ BROKEN[linux]=		libcpp won't build properly on FreeBSD hosts or genmatch segfaul
 [SOL10].CONFIGURE_ENV_ON=		LD=ld.bfd
 [SOL10].BROKEN_ON=			GCC10+ does not support solaris 10
 
+[MAMD64R30].DESCRIPTION=		Set target to MidnightBSD/AMD64 (Release 3.0)
+[MAMD64R30].BUILDRUN_DEPENDS_ON=	gnatcross-binutils:single:MAMD64_R30
+[MAMD64R30].MAKEFILE_ON=		OS_LABEL4VERS="[{{OPSYS}}/{{ARCH}} x MidnightBSD_30/AMD64]"
+					PREFIX={{LOCALBASE}}/MidnightBSD30_AMD64
+					SRPREFIX={{PREFIX}}/sysroot
+					XCCTARGET=x86_64-raven-freebsd${MIDNIGHTBSD_30}
```

Comment:
we really need to check this target.  I assume midnight does not identify as freebsd.  what is the target of native compiler?